src/lib/snarky_js_bindings/lib/snarky_js_bindings_lib.ml

```diff
@@ -1809,13 +1819,34 @@ let pickles_compile (choices : pickles_rule_js Js.js_array Js.t) =
           (Pickles.Verification_key.index key)
   end
 
+let pickles_digest, pickles_compile =
```

Comment:
Actually, @bkase could you revert this refactor which adds a level of indirection to pass in `constraint_constants` and `name` later? I would prefer if they were just hard-coded at the top level, and to have the final `pickles_digest` and `pickles_compile` declared at once.
Every level of indirection makes the code harder to understand, especially for newcomers, and the code is hard to understand already

Reply:
this is a good catch -- minimizing indirection is good here